### PR TITLE
DAOS-6326 Test: Remove large Transfer size for unaligned IO.

### PR DESCRIPTION
**Pull request by ravalsam**
SPDK does not support 1G of large transfer size in single
update so removing this option from the test.

Signed-off-by: Samir Raval <samir.raval@intel.com>

### CODE REVIEW

File: src/tests/ftest/io/unaligned_io.yaml [hunk 1]
```diff
@@ -25,5 +25,3 @@ datasize:
     - 300
     - 16000
     - 1048599
-    # Skip this large size until DAOS-2164 resolved.
-    #- 1073741824
```
Comment (rpadma2) [inline src/tests/ftest/io/unaligned_io.yaml:29]: Are you not planning to use cancelForTicket? I thought we should start using it...

**Review by rpadma2 — COMMENTED** [review]
Are you not p

File: src/tests/ftest/io/unaligned_io.yaml (hunk 1 above)
Comment (ravalsam) [reply src/tests/ftest/io/unaligned_io.yaml:29]: This is a large transfer size and it's not supported by SPDK so removing the tests.

Comment (rpadma2) [reply src/tests/ftest/io/unaligned_io.yaml:29]: ok.

**Review by rpadma2 — APPROVED** [review]

**Review by saurabhtandan — APPROVED** [review]
LGTM

**Review by NiuYawei — APPROVED** [review]

**Review by sylviachanoiyee — CHANGES_REQUESTED** [review]
If the test is no longer relevant, please remove from code. 
If you want to skip the test, waiting for a feature to be enabled or a bug to be fixed, please use cancelForTicket or skipForTicket.

**Comment by ravalsam** [comment]
> If the test is no longer relevant, please remove from code.
> If you want to skip the test, waiting for a feature to be enabled or a bug to be fixed, please use cancelForTicket or skipForTicket.

The test is a relevant but not that large 1G size so that size parameter has been removed from yaml file.

**Review by sylviachanoiyee — APPROVED** [review]

**Pull request merged**